delft3dworker/models.py

```diff
@@ -504,6 +495,9 @@ def publish_company(self, user):
         self.save()
 
     def publish_world(self, user):
+        if self.phase != self.phases.fin:
```

Comment:
Actually not: (as long as the Scene is in a finished state) the user can publish to world, no matter what state the run is in, i.e. "p", "c" or "w" (this last one is superfluous, but not incorrect - one can publish to world as often as one wishes).